actioncable/lib/assets/javascripts/action_cable/connection.coffee

```diff
@@ -60,11 +60,15 @@ class ActionCable.Connection
       switch type
         when message_types.confirmation
           @consumer.subscriptions.notify(identifier, "connected")
+          @dispatchEvent("connected", message, JSON.parse(identifier).channel)
         when message_types.rejection
           @consumer.subscriptions.reject(identifier)
         else
           @consumer.subscriptions.notify(identifier, "received", message)
 
+          if identifier isnt ActionCable.INTERNAL.identifiers.ping
```

Comment:
`message_types` contains `{confirmation: "confirm_subscription", rejection: "reject_subscription"}`. `ping` is not in message types because it is under `identifiers`